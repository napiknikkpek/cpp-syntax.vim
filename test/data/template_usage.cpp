Foo<T>::foo;
Foo<A, int, 3>::foo<A, int, 3, decltype(a), typename T::type>();
bucket<typename T::type>::foo();
bucket<(a > b)>::foo;
bucket<(a < b)>::foo;
bucket<decltype(a), int, decltype(hash(std::declval<T>(3)))>::foo();
bucket<typename decltype(a)::type>::foo();
