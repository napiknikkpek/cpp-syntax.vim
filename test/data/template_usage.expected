ttt.t...xxx.
ttt.t..zzz..o...fff.t..zzz..o..aaaaaaaa.x...aaaaaaaa.t..tttt....
tttttt.aaaaaaaa.t..tttt...fff...
tttttt..x...x....xxx.
tttttt..x...x....xxx.
tttttt.aaaaaaaa.x...zzz..aaaaaaaa.ffff.ttt..fffffff.t..o......fff...
tttttt.aaaaaaaa.aaaaaaaa.x...tttt...fff...
